astroquery/esa_hubble/tests/test_esa_hubble.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+
+@author: Javier Duran
+@contact: javier.duran@sciops.esa.int
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+Created on 13 Ago. 2018
+
+
+"""
+import pytest
+
+from astroquery.esa_hubble.core import ESAHubbleClass
+from astroquery.esa_hubble.tests.dummy_handler import DummyHandler
+from astroquery.esa_hubble.tests.dummy_tap_handler import DummyESAHubbleTapHandler
+from astropy import coordinates
+# from astropy.tests.helper import remote_data
+
+
+class TestESAHubble():
+    
+    def get_dummy_tap_handler(self):
+        parameterst = {}
+        parameterst['query'] = "select top 10 * from hsc_v2.hubble_sc2"
+        parameterst['output_file'] = "test2.vot"
+        parameterst['output_format'] = "votable"
+        parameterst['verbose'] = False
+        dummyTapHandler = DummyESAHubbleTapHandler("launch_job", parameterst)
+        return dummyTapHandler
+
+    def test_get_product(self):
+        parameters = {}
+        parameters['observation_id'] = "J6FL25S4Q"
+        parameters['calibration_level'] = "RAW"
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("get_product", parameters)
+        ehst = ESAHubbleClass(dummyHandler, self.get_dummy_tap_handler())
+        ehst.get_product("J6FL25S4Q", "RAW")
+        dummyHandler.check_call("get_product", parameters)
+
+    def test_get_artifact(self):
+        parameters = {}
+        parameters['artifact_id'] = "O5HKAX030_FLT.FITS"
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("get_artifact", parameters)
+        ehst = ESAHubbleClass(dummyHandler, self.get_dummy_tap_handler())
+        ehst.get_artifact("O5HKAX030_FLT.FITS")
```

Comment:
This line is triggering a failure for me locally:
`E           FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/_r/c4nkvmpn4vx6qc91qygz8h740000gr/T/astroquery-test-_awt8xh6/lib/python3.6/site-packages/astroquery/esa_hubble/tests/data/O5HKAX030_FLT.FITS'`
it may need to be added to the repository or added in the `setup_package.py` file